public/js/sticky-sidebar.js

```diff
@@ -0,0 +1,38 @@
+/* global $ */
```

Comment:
So I should revert the `import` / `require` stuff and do this suggestion instead? Because then the  `import` stuff becomes the more involved work later on?